src/selfplay/tournament.cc

```diff
@@ -67,8 +67,8 @@ void SelfPlayTournament::PopulateOptions(OptionsParser* options) {
       "multiplexing";
   options->Add<StringOption>(kNnBackendOptionsStr, "backend-opts");
   options->Add<BoolOption>(kVerboseThinkingStr, "verbose-thinking") = false;
-  options->Add<FloatOption>(kResignPlaythroughStr, 0.0f, 100.0f,
-                            "resign-playthrough") = 0.0f;
+  options->Add<FloatOption>(kResignPlaythruStr, 0.0f, 100.0f,
+                            "resign-playthru") = 0.0f;
```

Comment:
allow-resign-percentage
and 
"The percentage of games which are allowed to resign"
?

Reply:
That would change the value to 1-value, you'd have to take that up with Discord.